Kickstarter-iOS/Views/Cells/FindFriendsCell.swift

```diff
@@ -58,7 +58,7 @@ final class FindFriendsCell: UITableViewCell, ValueCell, NibLoading {
       .~ .ksr_text_dark_grey_400
     let arrowStyle = isDisabled ? disabledArrowStyle : settingsArrowViewStyle
     let backgroundColor: UIColor = isDisabled ? .ksr_grey_200 : .white
-    let titleLabelColor: UIColor = isDisabled ? .ksr_text_dark_grey_400 : .ksr_text_dark_grey_500
+    let titleLabelColor: UIColor = isDisabled ? .ksr_text_dark_grey_400 : .ksr_text_dark_grey_900
```

Comment:
So weren't we wanting to refer to `ksr_text_dark_grey_900` as `soft-black` now?

Reply:
That's a @dannyalright question. We used to have it in our code base couple commits back but then it was removed. I'd be happy to change it but right now there are `>100` uses of `ksr_text_dark_grey_900` in the codebase.

Reply:
Oh man! I didn't realise how out-of-whack our colour naming is compared to the Kickstarter style guide. Example: `ksr_text_dark_grey_400` is just `dark_grey_400` on the style guide, used for text sometimes and for fill at other times.

If `ksr_text_dark_grey_900` is equal to `#282828`, which is the same as `soft-black`, I say use the name `soft-black` everywhere.

If this is getting too complicated, let's just stick to the original task and do a _full_ color sweep separately. Thoughts?

BTW: huge fan of this PR. Thanks for putting so much detail in, @dusi.

Reply:
Thanks @dannyalright 

Agree...the refactoring (renaming) should be its own card and PR. If this looks good and passes AC we can go ahead with this for now. We should definitely put an effort into standardizing the color naming thorough KSR products